cmake/lz4.cmake

```diff
@@ -52,13 +52,6 @@ MACRO (FIND_SYSTEM_LZ4)
     NAMES lz4)
   IF (LZ4_INCLUDE_DIR AND LZ4_SYSTEM_LIBRARY)
     SET(SYSTEM_LZ4_FOUND 1)
-<<<<<<< HEAD
-    CHECK_LZ4_VERSION(${PATH_TO_LZ4})
```

Comment:
Why `CHECK_LZ4_VERSION()` removed?

Reply:
Good point. Wrong merge. It was our addition do cmake.